### ISSUE TITLE
Half Performace on MacOS

### ISSUE THREAD

**Issue by PaulMagos** [hieplpvip/ASUS-ZENBOOK-UX410-HACKINTOSH#96]
Hi , i've tried to change my CPUFriend profile many times but nothing 
changes, even by changing SMBIOS many times from 14,1 - 14,2  to 15,2 .
Can u help me ?

Asus Zenbook UX430UNR - 8 Gb Ram - i7-8550u (MX150 Disabled) 
BigSur 11.0.1  -  OC 0.6.3

GeekBench 5
CPU    Single Core      Multi Core
Win       991                   **3548**
Mac      933                   2183

GPU    OpenCL     
Win        **5829** 
Mac       ~3700-3900

[EFI.zip](https://github.com/hieplpvip/Asus-Zenbook-Hackintosh/files/5566373/EFI.zip)

**Comment by Ubsefor** [comment]
It could be that vendor (Asus) undervolts cpu or tweaks performance of CPU in any other way, so it doesn’t throttle too much under heavy loads… Also, I think you did notice the fans being quieter due to not spinning at speeds they spin in Windows. Also, windows has a better support for system ACPI, so… needless to say, the fans and such work better. Again, the cause of that is thermal throttling of your CPU. Just look at Intel Power Gadget for a moment, while running, for example, a Geekbench test. You will notice a drop of frequency after a short period of time (~5-10 seconds of Turbo-boost for me, lol) and that’s terrible.

TL;DR – we need to fix somehow CPU throttling, as the cooling system can’t keep up in Mac OS, for example, by undervolting (generally preferrable solution, as I do recall downvolting cpu in linux, which did use the fans in the same way as Mac OS) – either we need to make a custom cpu-friend patch with our own values at TPD/voltage/something else; Or we can fix DSDTs and add fan detection and managing to SMC and hope for the cooling system to be able to deal with the heat, but as far as I can tell, _hieplpvip_ didn’t manage to do that after several tries.

**Comment by VladBogdan** [comment]
Hei, is there any news on this issue ? I have the same configuration and at this point this is the only thing which doesn't work properly. Any answer regarding this is appreciated.

**Comment by Ubsefor** [comment]
> Hei, is there any news on this issue ? I have the same configuration and at this point this is the only thing which doesn't work properly. Any answer regarding this is appreciated.

Custom CPUFriend patch and downvolting using `VoltageShift` helps a bit.

**Comment by VladBogdan** [comment]
@Ubsefor What settings are you using for VoltageShift ?

**Comment by Ubsefor** [comment]
> @Ubsefor What settings are you using for VoltageShift ?

Its best for you to find optimal settings for yourself. Settings may vary for different systems. I suggest you start from -25mv for each and then (if the setup is stable) lower values for CPU and DRAM. 
I experienced hangups and freezes with GPU lower than -40mv or CPU/DRAM lower than -100mv, so keep that in mind. 

If the setup is not stable, just turn on SIP to prevent voltageShift from starting.
